src/nvim/mark.c

```diff
@@ -1499,12 +1501,13 @@ void copy_viminfo_marks(vir_T *virp, FILE *fp_out, int count, int eof, int flags
       if (load_marks) {
         if (line[1] != NUL) {
           int64_t lnum_64;
-          unsigned u;
+          unsigned int u;
```

Comment:
This is only stylistic, but why do you say `unsigned int` as opposed to `unsigned`? I don't do it myself for brevity, but then again it makes sense for consistency with types like `unsigned long` and `unsigned char`.